### ISSUE TITLE
API rate limit

### ISSUE THREAD

**Issue by jamiesong** [producthunt/producthunt-api#307]
Hi all, 

My research team is keen on using the Product Hunt API for an academic research paper on entrepreneurship, which helps uncover social biases faced by entrepreneurs on Product Hunt. 

However, we have observed that the rate limit for your API is quite restrictive. Would it be possible to either remove the rate limit or to significantly enhance its speed for the purposes of our academic project? 

Thank you very much.

**Comment by jtbg** [comment]
For future reference if others come across this issue, email in to `hello@producthunt.com`  with a description of the usage and what data you want to fetch etc 

> [If you require faster access without rate limit please contact us.](https://api.producthunt.com/v2/docs#:~:text=If%20you%20require%20faster%20access%20without%20rate%20limit%20please%20contact%20us.)

**Comment by jamiesong** [comment]
I have already emailed hello@producthunt.com a few months ago and they redirected me to this developer Github page. This issue has been pending for several months now and I would greatly appreciate it if you can increase the rate limit. Thanks.

**Comment by ClaytonSmith** [comment]
Did this ever get resolved? We're also trying to work with PH for larger scale research/hacker project

**Comment by jamiesong** [comment]
No, unfortunately this still has not been resolved. I would love to get any
insights if you were able to get some help from Product Hunt as they've
pretty silent.

**Comment by ClaytonSmith** [comment]
@jamiesong friend me on Discord @claytronymoose
